teams/migration/membership.json

```diff
@@ -0,0 +1,41 @@
+{
+    "_comment": [
+        "Items of lists in this file are in alphabetical order. Please keep it on modification."
+    ],
+    
+    "maintainers": [
+        "amyangfei",
+        "kennytm",
+        "lance6716"
+    ],
+
+    "committers": [
+        "3pointer",
+        "5kbpers",
+        "Ehco1996",
+        "GMHDBJD",
+        "IANTHEREAL",
+        "Leavrth",
+        "Little-Wallace",
+        "WangXiangUSTC",
+        "YuJuncen",
+        "csuzhangxc",
+        "glorv",
+        "holys",
+        "july2993",
+        "leoppro",
+        "lichunzhu",
+        "liuzix",
+        "lonng",
+        "overvenus",
+        "suzaku",
+        "zwj-coder"
+    ],
+
+    "reviewers": [
+        "gozssky",
+        "hi-rustin",
+        "iamxy",
+        "tiancaiamao"
```

Comment:
Could you add @ben1009 and @3AceShowHand ?

Reply:
updated

Reply:
Could you also add @asddongmen as a reviewer? Thanks!

Reply:
updated.